### PR TITLE
Added MODNet models

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: demos/background_subtraction_demo/python/background_subtraction_demo.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 sys.path.append(str(Path(__file__).resolve().parents[2] / 'common/python'))
 
-from openvino.model_zoo.model_api.models import MaskRCNNModel, OutputTransform, RESIZE_TYPES, YolactModel, ImageMattingWithBackground, VideoBackgroundMatting
+from openvino.model_zoo.model_api.models import MaskRCNNModel, OutputTransform, RESIZE_TYPES, YolactModel, ImageMattingWithBackground, VideoBackgroundMatting, PortraitBackgroundMatting
```
Comment (anzhella-pankratova) [inline demos/background_subtraction_demo/python/background_subtraction_demo.py:29]: This line becomes too long, let's split it into 2 lines

**Review by anzhella-pankratova — COMMENTED** [review]
Please, update the table with supported wrappers in the documentation for Python Model API with `PortraitBackgroundMatting`

File: demos/background_subtraction_demo/python/README.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ The demo application expects an instance segmentation or background matting mode
         * `fgr` with normalized in [0, 1] range foreground
         * `pha` with normalized in [0, 1] range alpha
         * recurrent outputs: `rr1`, `rr2`, `rr3`, `rr4`
+5. for portrait background matting models:
```
Comment (eaidova) [inline demos/background_subtraction_demo/python/README.md:40]: I would like rename for more general naming and move closer to 3

```suggestion
5. for image background matting models without trimap
```

File: models/public/modnet-photographic-portrait-matting/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+The `modnet-photographic-portrait-matting` model is a lightweight matting objective decomposition network (MODNet) for photographic portrait matting in real-time with a single input image with MobileNetV2 backbone. The model is pre-trained in PyTorch\* framework and converted to ONNX\* format.
+
+More details provided in the [paper](https://arxiv.org/abs/2011.11961) and [repository](https://github.com/ZHKKKe/MODNet).
+
+## Specification
+
+| Metric                          | Value              |
+|---------------------------------|--------------------|
+| Type                            | Background_matting |
```
Comment (eaidova) [inline models/public/modnet-photographic-portrait-matting/README.md:13]:
```suggestion
| Type                            | background_matting |
```

File: models/public/modnet-photographic-portrait-matting/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order: `BGR`.
+
+## Output
+
+### Original model
+
```
Comment (eaidova) [inline models/public/modnet-photographic-portrait-matting/README.md:58]: please add info about values range

File: models/public/modnet-photographic-portrait-matting/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Alpha matte with values in [0, 1] range. Name: `output` Shape: `1, 1, 512, 512`, format: `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+## Download a Model and Convert it into Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/model_tools/README.md) as shown in the examples below.
```
Comment (anzhella-pankratova) [inline models/public/modnet-photographic-portrait-matting/README.md:77]: Also, for the second model readme
```suggestion
## Download a Model and Convert it into OpenVINO™ IR Format

You can download models and if necessary convert them into OpenVINO™ IR format using the [Model Downloader and other automation tools](../../../tools/model_tools/README.md) as shown in the examples below.
```

**Review by anzhella-pankratova — APPROVED** [review]

**Review by Wovchena — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

File: demos/common/python/openvino/model_zoo/model_api/models/background_matting.py [hunk 6]
```diff
@@ -150,3 +150,36 @@ def postprocess(self, outputs, meta):
...
+    __model__ = 'Portrait-matting'
+
+    def __init__(self, model_adapter, configuration, preload=False):
+        super().__init__(model_adapter, configuration, preload)
+        self._check_io_number(1, 1)
+        self.output_blob_name = self._get_outputs()
+
+    @classmethod
+    def parameters(cls):
+        return super().parameters()
```
Comment (eaidova) [inline demos/common/python/openvino/model_zoo/model_api/models/background_matting.py:165]: no need in overriding method if there is no own realization
```suggestion
```

File: models/public/index.md [hunk 7]
```diff
@@ -33,6 +33,8 @@
    :caption: Background Matting Models
 
    omz_models_model_background_matting_mobilenetv2
+   omz_models_model_modnet_photographic_portrait_matting
```
Comment (eaidova) [inline models/public/index.md:36]: please align indention

**Comment by eaidova** [comment]
Jenkins please retry a build

File: demos/common/python/openvino/model_zoo/model_api/models/background_matting.py (hunk 6 above)
Comment (anzhella-pankratova) [reply demos/common/python/openvino/model_zoo/model_api/models/background_matting.py:165]: `VideoBackgroundMatting` and `ImageMattingWithBackground` wrappers has the same method, I would leave it as it is (just to be aligned with them)

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Pull request merged**